apps/api-extractor/src/schemas/api-extractor-template.json

```diff
@@ -215,7 +215,23 @@
      *
      * DEFAULT VALUE: "false"
      */
-    // "includeForgottenExports": false
+    // "includeForgottenExports": false,
+
+    /**
+     * The URL to the `<projectFolder>` token where the project's source code can be viewed on a website like GitHub or
```

Comment:
How does this relate to the `<projectFolder>` token?  The serialized paths are relative to the `<projectFolder>` value, but otherwise the `projectFolderUrl` is a URL not a disk path.

Reply:
The doc comment mentions the `<projectFolder>` token because this is the URL to the `projectFolder` folder on a source code viewing website like GitHub. Might this be less confusing if I use `projectFolder` instead of `<projectFolder>`? Maybe that's where the confusion is coming from?

Reply:
Updated this comment.